### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by yscialom** [puargs/ftopindexmaker#1]
Hello,

I found interest in your project and consider using it to build my personnal photo gallery.

I'm already a user of fgallery, and not unlike you find it lacking a higher-order feature.

If you're interested, and have time & motivation to give a refresh to your codebase, I'd be happy to assist you by providing it with a docker packaging so it can more easily be addded to people's stack. I might as well add a bit of automation with Github's actions too :)

Let me know. Happy coding.

**Comment by meliorisse** [comment]
Yep it still works! I use it for my family albums on a server I haven't updated dotnet. This project could absolutely use a refresh, I'd be happy to merge any updates :)
